src/task.c

```diff
@@ -644,6 +650,7 @@ task_enqueue(task_system *ts,
    }
 
    __sync_fetch_and_add(&group->current_waiting_tasks, 1);
+   __sync_fetch_and_add(&ts->ntasks_enqueued, 1);
```

Comment:
Metric to track how many tasks were enqueued.

This came very much in handy while running multi-threaded heavy insert workloads, to look closer into fg/bg-thread machinery is playing out.